CONTRIBUTING.md

```diff
@@ -20,4 +20,32 @@ reason about the proposed changes.
 Many of the tests require privileges to set resource limits and load eBPF code.
 The easiest way to obtain these is to run the tests with `sudo`:
 
-    sudo go test ./...
\ No newline at end of file
+    sudo go test ./...
+
+## Other platforms
+
+The main development platform that the project targets is `amd64`. However running
+the code and the examples should not be hard on any platform supported by `go` and
+`docker`. Here are some hints for `arm64`, adding other platforms should be similar.
+The suggested approach will always use `amd64` build tools, but allow to cross-build
+and use them with an emulation.
+
+#### Ubuntu `linux/arm64`
```

Comment:
So what commands would you usually run when developing on linux/arm64 then? `go test ./...`? `make`?